pkg/app/piped/cloudprovider/kubernetes/state.go

```diff
@@ -351,9 +351,11 @@ func determineJobHealth(obj *unstructured.Unstructured) (status model.Kubernetes
 			failed = true
 			completed = true
 			message = condition.Message
+			break
```

Comment:
Nice

Reply:
but without a label, it continues the for loop.

Reply:
Sorry, I couldn't understand what you mean.
Could you explain in more detail?

Reply:
I mean: https://go.dev/play/p/m5qecum0nVV

Reply:
This break just gets out from the switch statement. To break the wrapped For loop, you need to use Labels.

Reply:
Oh, it does make sense to me.
Thanks!